eslint.config.mjs

```diff
@@ -0,0 +1,339 @@
+import { fixupConfigRules, fixupPluginRules } from "@eslint/compat";
+import typescriptEslint from "@typescript-eslint/eslint-plugin";
+import prettier from "eslint-plugin-prettier";
+import tsParser from "@typescript-eslint/parser";
+import path from "node:path";
+import { fileURLToPath } from "node:url";
+import js from "@eslint/js";
+import { FlatCompat } from "@eslint/eslintrc";
+import { createRequire } from "module"
+
+// Workaround for https://github.com/anza-xyz/eslint-plugin-require-extensions/issues/18
+const require = createRequire(import.meta.url)
+const requireExtensions = require("eslint-plugin-require-extensions")
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+const compat = new FlatCompat({
+    baseDirectory: __dirname,
+    recommendedConfig: js.configs.recommended,
+    allConfig: js.configs.all
+});
+
+export default [{
+    ignores: [
+        "coverage/**/*",
```

Comment:
does `.eslintignore` no longer work?

Reply:
Yeah it's [deprecated](https://eslint.org/docs/latest/use/configure/ignore-deprecated)

Reply:
Yeah it's [deprecated](https://eslint.org/docs/latest/use/configure/ignore-deprecated)